applications/precompiled_pva/cpp/pva_unsharp_mask/c_model/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+project(pva_unsharp_mask)
+
+add_library(unsharp_mask_ref
+        unsharp_mask_ref.cpp)
+
+target_include_directories(unsharp_mask_ref
+        PUBLIC .)
```

Comment:
```suggestion
target_include_directories(unsharp_mask_ref
        PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>")
```

Reply:
Removed the CUPVA based algorithm source code and related documentation for now.
It will be added in the upcoming releases.